process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+
+#### Archiving Considerations
+
+When voting on a proposal to archive a project, TAC members may consider:
+* If the project has consistently met the Project Requirements
+* If the project still supports the OpenSSF mission
+* If the project's inactivity or inconsistent maintenance presents a user security risk
+* If the project's design or approach is no longer a recommended best practice
+
+#### Archive Process
+
+Archived projects are no longer in active development and are only archived after a TAC vote.
+
+What does archiving for an OpenSSF project mean?
+* OpenSSF will no longer provide budgetary, event, or marketing support for the project
+* Trademarks and domain names of archived projects are still hosted by the OpenSSF and the Linux Foundation
+* Project maintainers may request trademark transfers from the OpenSSF and the Linux Foundation
+* OpenSSF can provide services such as documentation updates to help transition users
+* Other OpenSSF marketing activities will no longer be provided for the project
+
+To archive a project:
+* A proposal must be put forth to the TAC or WG repo
+* The TAC or parent WG will inform the project maintainers, OpenSSF end user community and wider community of all archiving proposals
+* The proposal must remain open for at least 2 weeks of discussion after the maintainers are informed.
+* A vote must be finalized with 2/3 approval from the TAC or parent WG.
+
+
+## Submission Process
+
+### Project creation or change of lifecycle stage
+
+For initiating the creation of a new project or for requesting a change of a project's lifecycle stage, an application must be submitted to the TAC. To this end, a maintainer of the project (or any OpenSSF member in the case of Archived projects) creates a PR in this repository with the following changes:
+
+* A new file in the `project-lifecyle-documents` directory containing all information requested for a project creation or a lifecycle change review. This file must be based on the template for the respective lifecycle stage in the `templates` directory. The `PROJECT_NAME_` prefix of the template must be replaced by the project name.
+
+* Modification of the table listing all projects in the [README](../README.md) of this repository by either updating the status field of the project in the table to the intended new lifecycle stage or by adding the project to the table in case of a project creation request.
+
+The TAC members review the PR and upon approval according to voting criteria defined in the [OpenSSF charter](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf), the PR can be merged. Only after merging of the PR, the new project has been created and the new lifecycle stage is in effect.
```

Comment:
Suggest moving "only after merging of the PR" to the end of the sentence.

Reply:
Addressed in PR#117